diff --git a/grammar/liquid-html.ohm.js b/grammar/liquid-html.ohm.js
index 2f17cca99e059cf8cf59c0dcb87b6eef386325f4..3baa66387164d97632c542a5bdbecfac744b6ea5 100644
--- a/grammar/liquid-html.ohm.js
+++ b/grammar/liquid-html.ohm.js
@@ -283,6 +283,8 @@ Liquid <: Helpers {
     | "unless"
     | "tablerow"
     | "block"
+    | "apply"
+    | "macro"
     ) endOfIdentifier
 }
 
diff --git a/standalone.js b/standalone.js
index b154a86911f2dcf4090114928661059158296081..2d9e3ddf1e91507bc1bddc0df5f4a9bfa3e3b560 100644
--- a/standalone.js
+++ b/standalone.js
@@ -5138,6 +5138,8 @@ Liquid <: Helpers {
     | "unless"
     | "tablerow"
     | "block"
+    | "apply"
+    | "macro"
     ) endOfIdentifier
 }
 
