diff --git a/dist/blocks.d.ts b/dist/blocks.d.ts
index dab0228852d308d15091d2034570cb93b802d4df..ace683cdce4f83f4a357580aa03d4f31c7c49e91 100644
--- a/dist/blocks.d.ts
+++ b/dist/blocks.d.ts
@@ -80,7 +80,7 @@ declare enum SourceError {
 }
 interface SourceCodeProps {
     /** The language the syntax highlighter uses for your story’s code */
-    language?: SupportedLanguage;
+    language?: "twig" | SupportedLanguage;
     /** Use this to override the content of the source block. */
     code?: string;
     /** The formatter the syntax highlighter uses for your story’s code. */
diff --git a/dist/blocks.js b/dist/blocks.js
index f3f4343fbcf25251e75d799719d371b037e47604..1d7526ce9a744e3aaec852a39512419b1243c43a 100644
--- a/dist/blocks.js
+++ b/dist/blocks.js
@@ -6550,7 +6550,7 @@ var ArgsTable = /* @__PURE__ */ __name((props) => {
         ...common
       }
     ))
-  )), Object.entries(groups.sections).map(([category, section]) => React34.createElement(SectionRow, { key: category, label: category, level: "section", colSpan }, section.ungrouped.map((row) => React34.createElement(ArgRow, { key: row.key, row, arg: args && args[row.key], ...common })), Object.entries(section.subsections).map(([subcategory, subsection]) => React34.createElement(
+  )), Object.entries(groups.sections).sort((a, b)=>a[0].localeCompare(b[0])).map(([category, section]) => React34.createElement(SectionRow, { key: category, label: category, level: "section", colSpan }, section.ungrouped.map((row) => React34.createElement(ArgRow, { key: row.key, row, arg: args && args[row.key], ...common })), Object.entries(section.subsections).map(([subcategory, subsection]) => React34.createElement(
     SectionRow,
     {
       key: subcategory,
